parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -185,6 +211,18 @@ where
 			DescendOrigin(X1(PalletInstance(inbound_queue_pallet_index))),
 			// Change origin to the bridge.
 			UniversalOrigin(GlobalConsensus(network)),
+			// Refund the surplus execution weight to BH sovereign
+			SetAppendix(Xcm(vec![
+				RefundSurplus,
+				ClearOrigin,
+				DepositAsset {
+					assets: Wild(AllCounted(1u32)),
+					beneficiary: MultiLocation {
```

Comment:
The origin in this context should actually be `GlobalConsensus(Ethereum)`.

Also it better to be more specific, since other bridges are also using the BH sovereign account

Reply:
https://github.com/Snowfork/snowbridge/pull/1096/commits/ec4ce325c1b7bf49d5ea530863375c9c6827ee8a

Reply:
Sorry, I actually meant `recipient` instead of `origin`. The recipient of the surplus funds should be `GlobalConsensus(Ethereum)`. That way, some other bridge on BridgeHub can't come over and claim our surplus fees.